### PR TITLE
Sym file checker

### PR DESCRIPTION
**Pull request by tszwailam**
Check to determine whether sym files exists at the top level

### CODE REVIEW

File: code/dqe/symfilecheck.q [hunk 1]
```diff
@@ -0,0 +1,8 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  fullpath:.Q.dd[directory]filename;
+  $[not ()~key hsym fullpath;
```
Comment (jonathonmcmurray) [inline code/dqe/symfilecheck.q]: I think you can use `.os.Fex` here https://github.com/AquaQAnalytics/TorQ/blob/master/code/common/os.q#L4

```suggestion
  $[.os.Fex fullpath;
```

(could probably also lose the `fullpath` intermediate variable)

File: code/dqe/symfilecheck.q [hunk 2]
```diff
@@ -0,0 +1,8 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  (c;"sym file named ",(","sv string(), filename)," ",$[c:.os.Fex .Q.dd[directory]filename;"exists";"doesn't exists"])
+//  $[.os.Fex .Q.dd[directory]filename;
```
Comment (ThomasSmyth) [inline code/dqe/symfilecheck.q]: Remove commented code.

File: code/dqe/symfilecheck.q [hunk 3]
```diff
@@ -0,0 +1,8 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  (c;"sym file named ",(","sv string(), filename)," ",$[c:.os.Fex .Q.dd[directory]filename;"exists";"doesn't exists"])
```
Comment (ThomasSmyth) [inline code/dqe/symfilecheck.q]: Why is string stitching needed here? My assumption would be that filename is an atom.

File: code/dqe/symfilecheck.q [hunk 4]
```diff
@@ -0,0 +1,5 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
+symfilecheck:{[directory;filename]                     // if sym file exists
+  (c;"sym file named ",(string filename)," ",$[c:.os.Fex .Q.dd[directory]filename;"exists";"doesn't exists"])
```
Comment (ThomasSmyth) [inline code/dqe/symfilecheck.q]: Typo:
```suggestion
  (c;"sym file named ",(string filename)," ",$[c:.os.Fex .Q.dd[directory]filename;"exists";"doesn't exist"])
```

**Pull request merged**